tests/models/test_cls_pointnet++.py

```diff
@@ -0,0 +1,19 @@
+import hydra
+import pytest
+
+from tests.helpers.helpers import count_trainable_params
+from tests.helpers.run_if import RunIf
+
+test_cases = [("cls_pointnet++.yaml", 2, 6, 1_462_978), ("cls_pointnet++.yaml", 4, 8, 1_463_620)]
```

Comment:
I wonder if there's a cleaner way to do this, seems like pytest might be limited

Reply:
Yeahhh I tried to make it so that we could easily add the new models etc to the test cases